server/controllers/contest.js

```diff
@@ -177,22 +179,41 @@ exports.createContestCharge = asyncHandler(async (req, res, next) => {
   }
 });
 
-//This function is not complete yet it will be updated
-// The values are just for testing on stripe since we will
-// not be actually transferring money to any accounts
+/**
+ * Summary.
+ * After contest ends, owner of contest can select a winnning submission
+ * which will trigger this funciton. Since we are using fake stripe accounts
+ * for testing the stated fields below are needed for stripe to know this is
+ * only an immitation of what would happen in production
+ * @param id? string used to identify submission id
+ * @param user string used to identify user of submission
+ * @returns response object
+ *
+ */
 exports.selectContestWinner = asyncHandler(async (req, res, next) => {
+  const { id, user, price } = req.body;
+
+  let userInfo = await User.findById(user);
+  let updatedSubmission = await Submission.findByIdAndUpdate(
+    id,
+    {
+      is_winner: true,
+    },
+    { new: true }
+  );
+
   const account = await stripe.accounts.create({
     type: "custom",
     country: "CA",
-    email: "bob.rosen@example.com",
+    email: userInfo.email,
     business_type: "individual",
     business_profile: {
```

Comment:
Same here with using JavaScript conventions

Reply:
this was straight from stripe documentation i believe they are expecting the variables to be named that way, i cant find the exact place in the documentation where i saw this but it is similar to this https://stripe.com/docs/connect/testing